cmd/meroxa/turbine/core.go

```diff
@@ -0,0 +1,97 @@
+package turbine
```

Comment:
Do you want to put this in an internal package, so that its not exportable? 
e.g. `cmd/meroxa/turbine/internal`

Reply:
Sounds good. Internal just mean it can’t be imported as a package outside of CLI, it is importable internally